chapter-06/photo-share-client/src/AuthorizedUser.js

```diff
@@ -19,10 +19,10 @@ const CurrentUser = ({ name, avatar, logout }) =>
 
 const Me = ({ logout, requestCode, signingIn }) =>
     <Query query={ROOT_QUERY} fetchPolicy="cache-only">
-        {({ loading, data }) => data.me ?
+        {({ loading, data }) =>loading ?
```

Comment:
```suggestion
        {({ loading, data }) =>data?.me ?
```

I recommend Optional chaining